docsrc/installation.rst

```diff
@@ -80,38 +117,54 @@ to do so is via the ``install_cmdstan`` function
   virtual environment.
 
 
-  **Jupyter notebook users:**  CmdStanPy can display progress bars during sampling
-  as well as during the CmdStan build process; these are implemented using the
-  `tqdm <https://github.com/tqdm/tqdm>`_ package,  which uses the
-  `ipywidgets <https://ipywidgets.readthedocs.io/en/latest/index.html>`_ package
-  in order to update the browser display.  For further help on installation
-  and configuration, see
+  **Jupyter notebook users:**  If you intend to run CmdStanPy from within a Jupyter notebook,
+  you may need to install the
+  `ipywidgets <https://ipywidgets.readthedocs.io/en/latest/index.html>`_.
+  This will allow for progress bars implemented using the `tqdm <https://pypi.org/project/tqdm/>`_
+  to display properly in the browser.
+  For further help on Jupyter notebook installation and configuration , see
   `ipywidgets installation instructions <https://ipywidgets.readthedocs.io/en/latest/user_install.html#>`_
   and `this tqdm GitHub issue <https://github.com/tqdm/tqdm/issues/394#issuecomment-384743637>`_.
 
 
+.. _cmdstan-install:
+CmdStan Installation
+--------------------
+
+If you have installed CmdStanPy from PyPI or Github,
+**you must install CmdStan**.
+The recommended way to do so is via the ``install_cmdstan`` function
+:ref:`described below<install-cmdstan-fun>`.
+
+If you installed CmdStanPy with conda, CmdStan and the C++ toolchain,
+both CmdStan and the C++ toolchain are installed into directory ``$CONDA_PREFIX/bin``
+and you don't need to do any further installs.
 
-Installing CmdStan
-------------------
 
-Prerequisites
-^^^^^^^^^^^^^
+C++ Toolchain Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-CmdStanPy requires an installed C++ toolchain
-consisting of a modern C++ compiler and the GNU-Make utility.
+To compile a Stan program requires a modern C++ compiler and the GNU-Make build utility.
+These vary by operating system.
 
-+ Windows: CmdStanPy provides the function ``install_cxx_toolchain``
+**Linux** The required C++ compiler is ``g++ 4.9 3``.
+On most systems the GNU-Make utility is pre-installed and is the default ``make`` utility.
+There is usually a pre-installed C++ compiler as well, but not necessarily new enough.
 
-+ Linux: install g++ 4.9.3 or clang 6.0.  (GNU-Make is the default ``make`` utility)
+**MacOS** The Xcode and Xcode command line tools must be installed.  Xcode is available for free from the Mac App Store.
+To install the Xcode command line tools, run the shell command: ``xcode-select --install``.
+
+**Windows**  We recommend using the `RTools 4.0 <https://cran.r-project.org/bin/windows/Rtools/rtools40.html>`_ toolchain
+which contains a ``g++ 8`` compiler and ``Mingw``, the native Windows equivalent of the GNU-Make utility.
+If it is not found on your system, the :meth:`cmdstanpy.install_cmdstan` function will install it automatically.
```

Comment:
This is not exactly true - you need to pass `compiler=True`, which isn't quite automatic

Reply:
looks like command line version doesn't have `compiler=True` arg.  it should.

Reply:
@WardBrian could you do this in #579?

Reply:
It does have —compiler, but it only shows up on Windows

Reply:
great, thanks!